### DISCUSSION TITLE
Adding daily data to the energy dashboard

### DISCUSSION THREAD

**Discussion by Epimedium** [jobvk/Home-Assistant-Windcentrale#157] [Support]
Hi, it's a shame that live dat no longer exists. With the new update, is it also possible to show the daily production in the HA energy dashboard? If I want to add sensor.de_grote_geert_production_day_shares to the return to grid section it is not accepted. 
Best, Erik

**Comment by jobvk** [comment]
Hi @Epimedium,

The problem with `sensor.de_grote_geert_production_day_shares` is it shows the value of yesterday and the attributes shows the rest of the days see image below, and it only updates one every day. If you would add this sensor to the energy dashboard it would update the production value of yesterday completely into the 12-13 hour graph of today. So the graph would have wrong data. This is the reason I didn't add the support for the energy dashboard.

Best @jobvk 

![image](https://github.com/jobvk/Home-Assistant-Windcentrale/assets/32730202/903a417d-37b0-451a-a62e-315fafecd043)

**Comment by steenderen** [comment]
I tried to figure out a way to add the data to the energy dashboard. See https://steenderen.net/blog/winddelen-opbrengst-in-home-assistant-na-api-aanpassingen/ for all info (sorry in Dutch only). As discribed, I use the production day shares value, knowing this isn't real time, and I distribute it evenly over 24 hours as of the moment the value is published by the API. So yes, you've got a delay in data, but I think it is quite workable and better than having nothing in the energy dashboard.
